content/en/blog/becoming-a-todo-group-member-as-an-associate.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Becoming a TODO Group Member as a Linux Foundation Associate Member"
+author: todogroup
+date: 2020-08-27
+---
+
+Open source program offices have become a best practice for companies that rely on open source technologies. As open source program offices have spread, there has been increased interest in applying this best practice in non-corporate contexts. Cities, universities, and other non-corporate organizations are starting to see the benefits of creating a center of competency for open source.
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Members of the Linux Foundation are typically companies or corporations that pay annual membership fees. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions that wish to become Linux Foundation members and join the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group institution that meet the criteria for Associate Membership as spelled out in the [Linux Foundation's Bylaws.](https://www.linuxfoundation.org/bylaws/)
+
+> Associate Members shall be non-profit and government entities that engage in or support the production, manufacture, use, sale, or standardization of Linux or other open source-based technologies. An Associate Member shall have no voting rights, and shall meet such eligibility criteria and pay such fees (if any) as the Board of Directors may from time to time approve. An entity shall cease to be an Associate Member in the event of its resignation or expulsion from this corporation.```
+
+You can view the Linux Foundation Associate Membership application in the TODO Group's [Governance](https://github.com/todogroup/governance) repository, which also contains the TODO Charter and Membership Agreement. If you wish to join as an Associate Member, please email info@todogroup.org for more information.
```

Comment:
Just curious, does the LF have the associate membership app somewhere on the web? Seems odd that we have a copy in our repo.

Reply:
When I drafted the blog post, they did not. Checking it into our repo was my fix for that, so we could draft this guidance. Prior to that, when I asked where to find the application, the answer was "email and ask for it"

Reply:
Interesting. Thanks for the context.